src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -202,6 +201,11 @@ private void ComputeParameters()
                 addRefReadOnlyModifier: false,
                 diagnostics: diagnostics).Cast<SourceParameterSymbol, ParameterSymbol>();
 
+            foreach (var parameter in this.Syntax.ParameterList.Parameters)
+            {
+                WithTypeParametersBinder.ReportFieldOrValueContextualKeywordConflictIfAny(parameter, parameter.Identifier, diagnostics);
```

Comment:
>WithTypeParametersBinder.ReportFieldOrValueContextualKeywordConflictIfAny(parameter, parameter.Identifier, diagnostics); [](http://example.com/codeflow?start=16&length=120)

Consider moving this check into ```public static SourceParameterSymbol Create(``` which is called by ```ParameterHelpers.MakeParameters``` called above.

Reply:
If we want to limit the check to local function parameters, we should be able to check the owner there.

Reply:
The current location seems fine to me.